templates/thank-you.html

```diff
@@ -1,11 +1,38 @@
 {% extends 'layouts/_base.html' %}
+{% from "components/panel/_macro.njk" import onsPanel %}
 
 {% set page_title = _("We\'ve received your answers") + " - " + page_header.title %}
 
 {% block main %}
+  {% if schema_theme == "default" %}
+      <h1 data-qa="submission-successful-title">{{ _("Submission successful") }}</h1>
+      {% call onsPanel({
+            "spacious": true,
+            "type": "success",
+            "classes": "u-mb-s"
+          }) %}
+        <p>
+          {% if metadata.trad_as %}
+              {{ _("Your answers were submitted for <span>%(ru_name)s</span> (%(trading_as_name)s) on %(submitted_date_time)s",
+              ru_name = metadata.ru_name,
+              trading_as_name = metadata.trad_as,
+              submitted_date_time=metadata.submitted_time|format_datetime) }}
+          {% else %}
+              {{ _("Your answers were submitted for <span>%(ru_name)s</span> on %(submitted_date_time)s",
+              ru_name = metadata.ru_name,
+              submitted_date_time=metadata.submitted_time|format_datetime) }}
+          {% endif %}
+        </p>
+        <p>{{ _("Transaction ID") }}: <b>{{metadata.tx_id}}</b></p>
```

Comment:
The `tx_id` should be part of the string so that translators understand the context. This should be:

```
{{ _("Transaction ID: <b>{transaction_id}<b>").format(transaction_id=metadata.tx_id)
```